tests/unittests/threed/object_3d_tests.py

```
@@ -314,3 +314,467 @@ def test_traverse(self):
         self.assertListEqual(
             list(sub.traverse(include_self=False)), objects[2:4]
         )
+
+
+class TestObject3DRemove(unittest.TestCase):
```

Pylint (4.0.3)

[convention] Missing class docstring

(C0115)